services/web/server/src/simcore_service_webserver/projects/projects_exceptions.py

```diff
@@ -70,5 +70,5 @@ def __init__(self, user_id: UserID, project_uuid: ProjectID):
 class ProjectTooManyProjectOpened(ProjectsException):
     def __init__(self, max_num_projects: int):
         super().__init__(
-            f"You cannot open more than {max_num_projects} stud{'y' if max_num_projects == 1 else 'ies'} concurrently. Please close another project and retry."
+            f"You cannot open more than {max_num_projects} stud{'y' if max_num_projects == 1 else 'ies'} concurrently. Please, close another study and retry."

```

Comment:
Instead of `concurrently`, I'd use `at once`. Might be simpler to comprehend for the average user.